### DISCUSSION TITLE
range can't iterate over...

### DISCUSSION THREAD

**Discussion by twobwithkatt** [HEIGE-PCloud/DoIt#552] [Q&A] [answered]
Hi, i'm trying to add new categories in fontmatter, but it's showing this error
![image](https://user-images.githubusercontent.com/76549109/161274940-b3c7c40b-2f68-4462-bbdd-2305c5890bd1.png)
Can you help me fix this? thanks you first

**Comment by HEIGE-PCloud** [comment, marked as answer]
Use `categories: ["category 1", "category 2"]` instead of `categories: "category 1"`.

**Reply by twobwithkatt** [reply]
thanks y so much
